src/components/ResetMapButton.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div class="leaflet-control-zoom leaflet-bar">
+        <a
+            @click.prevent="resetView"
+            class="leaflet-control-zoom-in"
+            href="#"
+            title="Reset map"
+            aria-label="Reset map"
+            role="button"
+        >
+            <vue-feather type="home" size="20" style="margin-top: 4px; margin-right: 1px"></vue-feather>
```

Comment:
Why mix in feather icons when we're already using the md ones? The md home-outline icon seems fine, or do you not like the eaves? https://pictogrammers.com/library/mdi/icon/home-outline/

Reply:
Yeah I was away with the fairies when I made this decision!